The-PropertyEstimator-format.md

```diff
@@ -0,0 +1,290 @@
+# Property calculation toolkit
+
+Property calculation toolkit from the [Open Forcefield Consortium](http://openforcefield.org).
+
+This toolkit provides an API for storing, manipulating, and computing measured physical properties from simulation data.
+
+## API docs
+
+### Physical property measurements
+
+[**Physical property measurements**](https://en.wikipedia.org/wiki/Physical_property) are measured properties of a substance that provide some information about the physical parameters that define the interactions within the substance.
+A physical property is defined by a combination of:
+* A `Mixture` specifying the substance that the measurement was performed on
+* A `ThermodynamicState` specifying the thermodynamic conditions under which the measurement was performed
+* A `PhysicalProperty` is the physical property that was measured
+* A `MeasurementMethod` specifying the kind of measurement that was performed
+
+An example of each:
+* `Mixture`: a 0.8 mole fraction mixture of ethanol and water
+* `ThermodynamicState`: 298 kelvin, 1 atmosphere
+* `PhysicalProperty`: mass density
+* `MeasurementMethod`: vibrating tube method
+
+#### Physical substances
+
+We generally use the concept of a liquid or gas `Mixture`, which is a subclass of `Substance`.
+
+A simple liquid has only one component:
+```python
+liquid = Mixture()
+liquid.add_component('water')
+```
+
+A binary mixture has two components:
+```python
+binary_mixture = Mixture()
+binary_mixture.add_component('water', mole_fraction=0.2)
+binary_mixture.add_component('methanol') # assumed to be rest of mixture if no mole_fraction specified
+```
+
+A ternary mixture has three components:
+```python
+ternary_mixture = Mixture()
+ternary_mixture.add_component('ethanol', mole_fraction=0.2)
+ternary_mixture.add_component('methanol', mole_fraction=0.2)
+ternary_mixture.add_component('water')
+```
+
+The infinite dilution of one solute within a solvent or mixture is also specified as a `Mixture`, where the solute has zero mole fraction:
+```python
+infinite_dilution = Mixture()
+infinite_dilution.add_component('phenol', impurity=True) # infinite dilution; one copy only of the impurity
+infinite_dilution.add_component('water')
+```
+
+You can iterate over the components in a mixture:
+```python
+for component in mixture.components:
+    print (component.iupac_name, component.mole_fraction)
+```
+retrieve a component by name:
+```python
+component = mixture.components['ethanol']
+```
+or get the number of components in a mixture:
+```python
+ncomponents = mixture.ncomponents
```

Comment:
In other parts of the code, I think we've been naming properties like this `n_components`.

Reply:
For example, in the `topology` branch, check out the attribute names:
https://open-forcefield-toolkit.readthedocs.io/en/topology/api/generated/openforcefield.topology.Topology.html#openforcefield.topology.Topology